mizar/common/constants.py

```diff
@@ -65,6 +65,7 @@ class OBJ_STATUS:
     vpc_status_allocated = 'Alloc'
     vpc_status_ready = 'Ready'
     vpc_status_provisioned = obj_provisioned
+    vpc_status_error = 'Error'
```

Comment:
Could this be more specific? In this case, duplicate VNI.

Reply:
Updated to duplicated error